### PR TITLE
Drop old TOF compressor

### PR DESCRIPTION
**Pull request by ktf**
Drop old TOF compressor

### CODE REVIEW

**Comment by ktf** [comment]
@noferini any objections to this?

**Comment by noferini** [comment]
Hi @ktf ,
just a minor comment.
I would not remove the --old option for the moment in the workflow. I will mark it as obsolete (no effect).
We should remove it once we are sure that is no longer used in our flp workflows

File: Detectors/TOF/compression/src/tof-compressor.cxx [hunk 1]
```diff
@@ -45,7 +44,6 @@ void customize(std::vector<ConfigParamSpec>& workflowOptions)
   workflowOptions.push_back(paranoid);
   workflowOptions.push_back(ignoreStf);
   workflowOptions.push_back(payloadlim);
-  workflowOptions.emplace_back(ConfigParamSpec{"old", VariantType::Bool, false, {"use the non-DPL version of the compressor"}});
```
Comment (noferini) [inline Detectors/TOF/compression/src/tof-compressor.cxx]: Let's keep ths for the moment

**Comment by ktf** [comment]
You mean like this?

File: Detectors/TOF/compression/src/tof-compressor.cxx [hunk 2]
```diff
@@ -62,6 +61,9 @@ WorkflowSpec defineDataProcessing(ConfigContext const& cfgc)
   auto paranoid = cfgc.options().get<bool>("tof-compressor-paranoid");
   auto ignoreStf = cfgc.options().get<bool>("ignore-dist-stf");
   auto old = cfgc.options().get<bool>("old");
+  if (old) {
+    LOGP(error, "--old option is not actually supported, using the new version");
+  }
```
Comment (ktf) [inline Detectors/TOF/compression/src/tof-compressor.cxx:66]: @noferini something like this?

**Comment by ktf** [comment]
@noferini Ping.

**Review by noferini — APPROVED** [review]
It is fine
Thanks
Francesco

**Pull request merged**